ttnn/cpp/ttnn/operations/conv2d/conv2d.cpp

```diff
@@ -89,7 +89,7 @@ ParallelConfig determine_parallel_config(
             uint32_t total_cores_for_channels =
                 block_shard_orientation == ShardOrientation::COL_MAJOR ? device_grid_size[1] : device_grid_size[0];
             uint32_t num_cores_channels = find_closest_common_largest_divisor(
-                conv_out_2d_matrix_width_ntiles, std::ceil((double)input_channels / (double)32), total_cores_for_channels);
+                    conv_out_2d_matrix_width_ntiles, is_out_tiled ? std::ceil((double)input_channels / (double)32) : input_channels, total_cores_for_channels);
```

Comment:
shouldn't `conv_out_2d_matrix_width_ntiles` by multiplied by tile width for non-tiled case?